index.js

```diff
@@ -1,4 +1,10 @@
 'use strict'
+const createNamespace = require('continuation-local-storage').createNamespace
+const namespace = createNamespace('requestNamespace')
+
+global.Promise=require("bluebird")
```

Comment:
use `'` — single quotes, linter requirement is to use `"` only for interpolation, e.g. `"${}"`
fix spaces around `=`